addon/templates/components/frost-multi-select.hbs

```diff
@@ -17,7 +17,7 @@
     </div>
     <div class='multi-status'>
       <span class='number-selected'>{{selected.length}} selected</span>
-      <a href='#' class='clear' tabIndex={-1} onClick={{action "clearSelection"}}>Clear</a>
+      <a class='clear' tabIndex={-1} onClick={{action "clearSelection"}}>Clear</a>
```

Comment:
In the demo app, clearing doesn't update the URL when there's something to clear, but does when there isn't... If you're sure removing the `href` attribute doesn't have any negative side-effects, I guess I'm OK with it. I've never really used the `href="#"` myself, but another option (if it is needed for something) is to `e.preventDefault()` in the on-click (at least with straight javascript that works, not sure in the ember world).